packages/fake-driver/test/e2e/alert-tests.js

```diff
@@ -17,15 +17,15 @@ function alertTests () {
       await driver.dismissAlert().should.eventually.be.rejectedWith({code: 27});
     });
     it('should get text of an alert', async function () {
-      await (await driver.$('#AlertButton')).click();
+      await ((await driver.$('#AlertButton'))).click();
```

Comment:
syntactically this change seems unnecessary. can you explain what it's doing?

Reply:
that'd be a search/replace gone awry